### PR TITLE
Add Stale Issue Auto-Close Github Action

### PR DESCRIPTION
**Pull request by michhyun1**
**Issue #:**

**Description of changes:**

Automatically close stale issues. OSS has approved.

https://github.com/marketplace/actions/close-stale-issues

**Testing:**
Needs to be merged in order to work.

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*
No

**Checklist:**

1. Have you successfully run `npm run build:release` locally?
yes

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
no

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
no

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/close-stale-issues.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          days-before-issue-stale: 30
```
Comment (hensmi-amazon) [inline .github/workflows/close-stale-issues.yml:17]: Does 30 seem...a bit aggressive? Do we really have that many issues/prs that we can't do this sort of triage manually and not risking closing relevant issues?

File: .github/workflows/close-stale-issues.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+          days-before-close: 7
+          stale-issue-message: 'This issue has not had any updates in 30 days. If there are no updates/comments within 7 days, this issue will be closed.'
+          stale-pr-message: 'This pull request has not had any updates in 30 days. If there are no updates/comments within 7 days, this issue will be closed.'
+          close-issue-message: 'This issue has been closed due to inactivity. If your issue has not been resolved, please open a new issue and link this issue.'
+          close-pr-message: 'This issue has been closed due to inactivity. Please open a new pull request if needed.'
+          stale-issue-label: 'Stale'
+          close-issue-label: 'Auto-Closed'
+          stale-pr-label: 'Stale'
+          close-pr-label: 'Auto-Closed'
+          exempt-issue-labels: 'Browser Bug,Bug,dependencies,Documentation,feature-request,Triaged'
```
Comment (ltrung) [inline .github/workflows/close-stale-issues.yml:28]: Can we just do only auto close `Need More Information` for now?

Comment (michhyun1) [reply .github/workflows/close-stale-issues.yml:28]: mm not so sure about that - the `Needs more info` label isnt removed from the issue when a customer replies, so we might have an issue where we mark the label, the customer replies, but the label is still on the issue. Then this tool might close the ticket if we dont reply within the specified time.

File: .github/workflows/close-stale-issues.yml (hunk 1 above)
Comment (hensmi-amazon) [reply .github/workflows/close-stale-issues.yml:17]: Oh I didn't realize all the labels in `exempt-issue-labels`. I think this is probably fine.

Comment (michhyun1) [reply .github/workflows/close-stale-issues.yml:17]: Right - if you dont want to close the issue, then just apply a label to it. We can adjsut the list of exempt labels. This is just a "POC"

File: .github/workflows/close-stale-issues.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+          days-before-close: 14
+          stale-issue-message: 'This issue has not had any updates in 60 days. If there are no updates/comments within 14 days, this issue will be closed.'
+          stale-pr-message: 'This pull request has not had any updates in 60 days. If there are no updates/comments within 14 days, this pull request will be closed.'
+          close-issue-message: 'This issue has been closed due to inactivity. If your issue has not been resolved, please open a new issue and link this issue.'
+          close-pr-message: 'This issue has been closed due to inactivity. Please open a new pull request if needed.'
+          stale-issue-label: 'Stale'
+          close-issue-label: 'Auto-Closed'
+          stale-pr-label: 'Stale'
+          close-pr-label: 'Auto-Closed'
+          exempt-issue-labels: 'Browser Bug,Bug,dependencies,Documentation,feature-request,Triaged'
```
Comment (nainkunal933) [inline .github/workflows/close-stale-issues.yml:28]: @michhyun1 Can we make the `audio` `video` labels exempt as well or any label associated with other teams? We do not want to auto close issues that might belong to different team.

**Comment by michhyun1** [comment]
@ltrung sorry didnt get a chance to talk about this more - any feedback?

File: .github/workflows/close-stale-issues.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+          operations-per-run: 30
+          remove-stale-when-updated: true
```
Comment (xuesichao) [inline .github/workflows/close-stale-issues.yml:30]: nit: add new line end of file

**Pull request closed without merging**